docs/applications/pyserver/components-overview.rst

```diff
@@ -1,6 +1,12 @@
+
 Components overview
 --------------------
 
+This section will give you an overview of the classes used to build the uRaiden proxy. It should help keeping track of functionality of the proxy server.
+The Illustrations are inspired by UML even though they might not be exact standard.
+There is an overview for the Channel Manager class
```

Comment:
should this sentence be in there?

Reply:
We can leave it there for the time being.

Reply:
This is obsolete since the new updates from #462